json/test_resources/example.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I feel those may change and should probably not be used for tests. In any case I think it's better if the package owns its test resources